nautobot/dcim/views.py

```diff
@@ -1931,7 +1931,6 @@ def get_extra_context(self, request, instance):
                 "tags",
             )
             .select_related("lag", "cable")
-            .order_by("_name")
         )
         interface_table = tables.DeviceModuleInterfaceTable(data=interfaces, user=request.user, orderable=False)
         if VirtualChassis.objects.filter(master=instance).exists():
```

Comment:
Can we add an `interface_table.columns.show("role")` around here to address #5841?